core/src/main/java/de/danoeh/antennapod/core/preferences/UsageStatistics.java

```diff
@@ -49,10 +50,15 @@ public static void logAction(StatsAction action) {
     }
 
     public static boolean hasSignificantBiasTo(StatsAction action) {
-        final float movingAverage = prefs.getFloat(action.type, 0.5f);
-        final long askAfter = prefs.getLong(action.type + SUFFIX_HIDDEN_UNTIL, 0);
-        return Math.abs(action.value - movingAverage) < MOVING_AVERAGE_BIAS_THRESHOLD
-                && Calendar.getInstance().getTimeInMillis() > askAfter;
+        final boolean dontAsk = prefs.getBoolean(action.type + SUFFIX_HIDDEN_FOREVER, false);
+        if (dontAsk) {
+            return false;
+        } else {
+            final float movingAverage = prefs.getFloat(action.type, 0.5f);
+            final long askAfter = prefs.getLong(action.type + SUFFIX_HIDDEN_UNTIL, 0);
+            return Math.abs(action.value - movingAverage) < MOVING_AVERAGE_BIAS_THRESHOLD
+                    && Calendar.getInstance().getTimeInMillis() > askAfter;
+        }
```

Comment:
I think doing an early return would make it a bit easier to read because it reduces nesting.

```suggestion
        if (prefs.getBoolean(action.type + SUFFIX_HIDDEN_FOREVER, false)) {
            return false;
        }
        final float movingAverage = prefs.getFloat(action.type, 0.5f);
        final long askAfter = prefs.getLong(action.type + SUFFIX_HIDDEN_UNTIL, 0);
        return Math.abs(action.value - movingAverage) < MOVING_AVERAGE_BIAS_THRESHOLD
                && Calendar.getInstance().getTimeInMillis() > askAfter;
```